[FEATURE] Tmp retours ig module biais ind

JTEPIX opened the pull request:
## 🍂 Problème

<!-- Décrivez ici le besoin ou l'intention couvert par cette Pull Request. -->

## 🌰 Proposition

<!-- Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés. -->

## 🍁 Remarques

<!-- Des infos supplémentaires, trucs et astuces ? -->

## 🪵 Pour tester

https://app-pr13936.review.pix.fr/modules/tmp-ia-bias-ind

JTEPIX committed:
Update IAGenBiais_IND.json

Modifications liés aux retours de l'inspection générale

JTEPIX committed:
Update IAGenBiais_IND.json

JTEPIX committed:
coquille QAB

JTEPIX committed:
alternative textuelle

JTEPIX committed:
modif découverte "Captain Amercia" vers "Jus de pomme"

pix-bot-github commented:
Choisir les applications à déployer :

- [x] [API](https://api-pr13936.review.pix.fr/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr13936/environment) <!-- pix-api-review -->
- [x] [App (.fr)](https://app-pr13936.review.pix.fr) – [App (.org)](https://app-pr13936.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-app-review-pr13936/environment) <!-- pix-app-review -->
- [ ] [Orga (.fr)](https://orga-pr13936.review.pix.fr) – [Orga (.org)](https://orga-pr13936.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-orga-review-pr13936/environment) <!-- pix-orga-review -->
- [ ] [Certif (.fr)](https://certif-pr13936.review.pix.fr) – [Certif (.org)](https://certif-pr13936.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-certif-review-pr13936/environment) <!-- pix-certif-review -->
- [ ] [Junior](https://junior-pr13936.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-junior-review-pr13936/environment) <!-- pix-junior-review -->
- [ ] [Admin](https://admin-pr13936.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-admin-review-pr13936/environment) <!-- pix-admin-review -->
- [ ] [API MaDDo](https://pix-api-maddo-review-pr13936.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr13936/environment) <!-- pix-api-maddo-review -->
- [ ] [Audit Logger](https://pix-audit-logger-review-pr13936.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr13936/environment) <!-- pix-audit-logger-review -->

> [!IMPORTANT]
> N'oubliez pas de déployer l'API pour pouvoir accéder aux fronts et/ou à l’API MaDDo.

JTEPIX committed:
modification exemple dernier exemple découverte

Suite aux retours PYO et IG, modification de l'exemple de Svetlana (hotesse de l'air) vers un exemple sur l'orientation (genré)

JTEPIX committed:
Précision feedback Activité caroussel (sport genré)

Précision du feedback suite à un retour de Jeff

JTEPIX committed:
modification dernière activité (biais de confirmation bitcoin)

transformation QCM en 2 steps de QCU suite à l'analyse des stats

JTEPIX committed:
retours texte validés en comité qualité

JTEPIX committed:
ajout QCU biais intro

post comité qualité

JTEPIX committed:
remis carousel disparu

JTEPIX committed:
réparé doublon uuid

JTEPIX committed:
Update IAGenBiais_IND.json

JTEPIX committed:
Update IAGenBiais_IND.json

JTEPIX committed:
Update IAGenBiais_IND.json

pix-service commented:
:tada: This PR is included in version 5.246.0 :tada:

The release is available on [GitHub release](https://github.com/1024pix/pix/releases/tag/v5.246.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
